templates/web/setup/step2.html.twig

```diff
@@ -637,6 +633,71 @@
     </div>
 
 
+    <div class="steps">
+            <div class="wrapper">
+                <div class="PNB10 wrapper_left">
+                    <div class="PNB10 section_title">
+                        <h2>4/5
+                            {{ 'SearchEngine settings' | trans }}
+                        </h2>
+                    </div>
+                    <div class="PNB10" style="top:50px;overflow-x:hidden;">
+                        <table class="main_content_wrapper">
+                            <tr>
+                                <td>
+                                    <table class="main_content_table">
+                                        <tr>
+                                            <td><label>{{ 'ElasticSearch server host' | trans }}</label></td>
+                                            <td><input class="" type="text" name="elasticsearch_settings[host]" value="" /></td>
+                                        </tr>
+                                        <tr>
+                                            <td><label>{{ 'ElasticSearch service port' | trans }}</label></td>
+                                            <td><input class="" type="number" name="elasticsearch_settings[port]" value="" /></td>
+                                        </tr>
+                                        <tr>
+                                            <td><label>{{ 'Use an existing index' | trans }}</label></td>
+                                            <td><input class="" type="checkbox" name="elasticsearch_settings[isexist]" value="" /></td>
```

Comment:
"elasticsearch_settings[use_existing_index]" instead of "elasticsearch_settings[use_existing_index]"